src/utils/codec.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file Codec
+ * @name codec.ts<utils>
+ * @description |
+ *
+ *    Encoding base to emoji and converting to any formats.
+ *
+ *    Every time you import this file will auto-calcuting all emojis
+ *    instead of a big constants file.
+ */
+export const ALL_EMOJIS: string[] = (function () {
+    const range: number[][] = [
```

Comment:
`[number, number][]`?

Reply:
These are code points in the decimal system and their lengths actually(hard to read)(for calculating the emojis)...we should remove this function if we use a constant file.

Reply:
ok

Reply:
`[number, number][]` this coding style? I'll give it a try.

Reply:
We prefer a more precise type definition if it is possible